attributes/config.rb

```diff
@@ -44,3 +44,25 @@
   default['cdap']['cdap_site']['gateway.server.port'] = '10000'
   default['cdap']['cdap_site']['gateway.memory.mb'] = '512'
 end
+
+# HDP 2.2+ support
+hdp_version =
+  case node['hadoop']['distribution_version']
+  when '2.2.0.0'
+    '2.2.0.0-2041'
```

Comment:
Yeah.. they're iterations. They're internal to HDP and aren't really documented, anywhere. I pull them from the directories on-disk after installing or from the package names...

```
# yum list installed | grep hadoop | grep -v ambari | head -n 1
hadoop_2_2_6_0_2800.x86_64               2.6.0.2.2.6.0-2800.el6        @HDP-2.2
```

The first three parts are the actual Hadoop version, 2.6.0... the rest is the HDP version and matches for all packages in a given HDP release.